lib/kubernetes-deploy/kubernetes_resource/pod_set_base.rb

```diff
@@ -2,10 +2,12 @@
 module KubernetesDeploy
   class PodSetBase < KubernetesResource
     def failure_message
+      "Pods in #{name} #{self.class.name.demodulize} are failing. " +
```

Comment:
Example of Daemonset with this message:

![image](https://user-images.githubusercontent.com/4789493/33353178-77a09ef2-d47a-11e7-847e-6f8ce82b9015.png)